### PR TITLE
[INLONG-4358][Manager] Support SqlServer source

### LINKED ISSUE #4358: [Feature][Manager] support sqlserver source
### Description

[Feature][Manager] support sqlserver source 

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Title Name: [INLONG-4358][Manager]Support sqlserver source

Fixes #4358 

### Verifying this change

- [X] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes)
  - If yes, how is the feature documented? (JavaDocs)

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/source/sqlserver/SqlServerSourceDTO.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                .port(request.getPort())
+                .database(request.getDatabase())
+                .schemaName(request.getSchemaName())
+                .tableName(request.getTableName())
+                .serverTimezone(request.getServerTimezone())
+                .allMigration(request.isAllMigration())
+                .primaryKey(request.getPrimaryKey())
+                .build();
+    }
+
```
Comment (GanfengTan) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/source/sqlserver/SqlServerSourceDTO.java:90]: Please add java doc

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/SqlServerSource.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+public class SqlServerSource extends StreamSource {
+
+    @ApiModelProperty(value = "DataSource type", required = true)
+    private SourceType sourceType = SourceType.SQLSERVER;
+
+    @ApiModelProperty("SyncType for SqlServerSource")
+    private SyncType syncType;
+
+    @ApiModelProperty("Data format type")
+    private DataFormat dataFormat = DataFormat.NONE;
```
Comment (gong) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/SqlServerSource.java:48]: This field should not be need.

**Review by gong — COMMENTED** [review]
please resolve conflict

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/SqlServerSource.java (hunk 2 above)
Comment (LvJiancheng) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/SqlServerSource.java:48]: This field extends from parent class

**Review by gong — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by GanfengTan — APPROVED** [review]

**Pull request merged**